clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -47,3 +47,8 @@ spec:
             storage: {{MultiplyInt 10 (AddInt 1 (DivideInt .Nodes 1000))}}Gi
   query:
     maxSamples: 100000000
+  {{if and .WINDOWS_NODE_TEST (eq .PROVIDER "gce")}}
```

Comment:
Let's keep the yamls as simple as possible. It would be better to have a dedicated variable controlling whether scraping windows nodes should be enabled or not. Something like:
```
{{if  .PROMETHEUS_SCRAPE_WINDOWS_NODES}}
...
{{end}}
```

This is a common pattern, e.g. see https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/prometheus/manifests/master-ip/master-endpoints.yaml

Reply:
I see, fixed